yarn.lock

```diff
@@ -934,14 +951,19 @@ ansi-regex@^5.0.1:
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-5.0.1.tgz#082cb2c89c9fe8659a311a53bd6a4dc5301db304"
   integrity sha512-quJQXlTSUGL2LH9SUXo8VwsY4soanhgo6LNSm84E1LBcE8s3O0wpdiRzyR9z/ZZJMlMWv37qOOb9pdJlMUEKFQ==
 
+ansi-regex@^6.0.1:
```

Comment:
info Reasons this module exists
   - Hoisted from "strip-ansi#ansi-regex"
   - Hoisted from "tsup#sucrase#glob#jackspeak#@isaacs#cliui#strip-ansi-cjs#ansi-regex"
   - "tsup#sucrase#glob#jackspeak#@isaacs#cliui#strip-ansi" depends on it
   - Hoisted from "tsup#sucrase#glob#jackspeak#@isaacs#cliui#strip-ansi#ansi-regex"
   - "tsup#sucrase#glob#jackspeak#@isaacs#cliui#wrap-ansi#strip-ansi" depends on it
   - Hoisted from "tsup#sucrase#glob#jackspeak#@isaacs#cliui#wrap-ansi#strip-ansi#ansi-regex"